src/main/java/touro/simondrums/UserListenerEvents.java

```diff
@@ -0,0 +1,49 @@
+package touro.simondrums;
+
+import javax.swing.*;
+import java.util.ArrayList;
+
+public class UserListenerEvents {
+    SimonGame game;
+    AudioPlayer audioPlayer = new AudioPlayer();
+    JLabel highScore;
+    JLabel currentScore;
+    CompListenerEvents compListenerEvent;
+    boolean gameIsPlaying;
+
+    public UserListenerEvents(SimonGame game, JLabel highScore, JLabel currentScore) {
+        this.game = game;
+        this.highScore = highScore;
+        this.currentScore = currentScore;
+        this.compListenerEvent = new CompListenerEvents(game, audioPlayer);
+    }
+
+    public void newGame() {
+        game.newGame();
+        gameIsPlaying = true;
+        currentScore.setText("Current Score: " + game.getCurrentScore());        compListenerEvent.playSequence();
```

Comment:
I'm guessing these two should have their own lines

Reply:
also if you're starting a new game, you could probs just say "current score: 0", but thats potentially a style choice